ros2pkg/ros2pkg/resource/ament_python/test_copyright.py.em

```diff
@@ -0,0 +1,23 @@
+# Copyright 2015 Open Source Robotics Foundation, Inc.
```

Comment:
should we template these years / copyright owners? for all of these python source files

Reply:
For context, I copied the test_* files from [here](https://github.com/ros2/demos/tree/master/demo_nodes_py/test) with the only change being adding an extra `@` symbol in front of the decorators so Empy wouldn't try to expand them.

Reply:
These test templates can stay as is since they are copied from somewhere with that year and copyright name. For other Python modules being generated the current year and the actual copyright holder (calling the generation script) would be great.

Reply:
@dirk-thomas The current generated files are fairly barren, matching the generated files for C++. Do you think it's worth a separate PR to add this to the C++ generated files as well?

Reply:
It's definitely outside the scope of this PR, and I think we can expect in this tooling for the user to pick their own license. Not every user is going to license their library as Apache 2, so I don't think we need to add any licensing header to the template files for user code in this PR.